scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.option('--portal', required=True, type=str,
```

Comment:
The full command is `qiita db portal add-analyses --portal lolercoaster --analyses 1,2,3` So not sure if name would clarify anything in this case. @biocore/qiita-dev thoughts?